GitUI/CommandsDialogs/BrowseDialog/FormOpenDirectory.cs

```diff
@@ -121,14 +121,13 @@ private void folderGoUpbutton_Click(object sender, EventArgs e)
 
         private void _NO_TRANSLATE_Directory_TextChanged(object sender, EventArgs e)
         {
-            try
```

Comment:
Restored the try-catch clause and disabled the folderGoUpbutton into catch block.
Thanks.